exam-01/section-2-3.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
```

Comment:
Section 2: `15/15`
Section 3: `15/20`

Good work creating a decent page, however, if we have decided to use Bootstrap, we can simplify a lot more by using purely Bootstrap CSS